product-service/src/constants.ts

```diff
@@ -0,0 +1,9 @@
+export const headers = {
+    'Content-Type': 'application/json',
+    'Access-Control-Allow-Methods': '*',
+    'Access-Control-Allow-Origin': '*',
+};
+
+export const ERROR_MESSAGES = {
+    NOT_FOUND: 'Product not found. Please try another product id',
+};
```

Comment:
Very often enum is used in this case.
Look here: https://www.typescriptlang.org/docs/handbook/enums.html